### DISCUSSION TITLE
Server code 204

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#597] [General]
Hi,

I saw a users in the log with a code of 204. I  searched for this code and it says:

HTTP Status 204 is not an error. It just means the request was successful with empty response body as nothing needs to be returned.

What is he trying to do?

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
i cannot say from a response code what a user wants to do.
The code is the response of HFS, not the user.

**Reply by PH5HP** [reply]
Hi Rejetto,

Ok. I thought the word 'OPTION' would tell you something.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
option is the "method", i guess.
still not much to say.
maybe if i knew all about the request...
